cognite/neat/rules/models/entities/_constants.py

```diff
@@ -56,10 +57,6 @@ class EntityTypes(StrEnum):
 
```

Comment:
My thinking is that if it is only used in one module/subpackage, it can kept there. If it starts used all over neat, then move it to the utils.regex_patterns